commons/all/src/main/java/org/infinispan/commons/marshall/Ids.java

```diff
@@ -50,7 +50,7 @@
    int XID_DEADLOCK_DETECTING_GLOBAL_TRANSACTION = 33;
    int XID_GLOBAL_TRANSACTION = 34;
 
-   //int IN_DOUBT_TX_INFO = 35;
+   int IN_DOUBT_TX_INFO = 35;
```

Comment:
Looks like we can revert these two.

Reply:
`InDoubtTxInfo` and `XidImpl` must use the Externalizer and not Protostream.
That's why I'm reverting the previous changes.

Reply:
Oh sorry I read the changes in inverse :)